lbz/resource.py

```diff
@@ -135,14 +134,22 @@ class CORSResource(Resource):
         "X-Amz-Security-Token",
     ]
 
-    def __init__(self, event: dict, methods: List[str], origins: List[str] = None):
+    def __init__(
+        self,
+        event: dict,
+        methods: List[str],
+        origins: List[str] = None,
+        cors_headers: List[str] = None,
```

Comment:
Still, I see some inconsistency here: we have already `methods` and `origins` in this class which in the end means allowed values. So, following this pattern, I think I would name this simply `headers` (meaning: allowed headers) 😉

Reply:
Or, if we want to provide a new pattern, TODO would be welcome to adjust the rest of the arguments in the near future too.